xwiki-platform-core/xwiki-platform-ckeditor/xwiki-platform-ckeditor-test/xwiki-platform-ckeditor-test-docker/src/test/it/org/xwiki/ckeditor/test/ui/ImageIT.java

```diff
@@ -65,7 +65,14 @@
  * @version $Id$
  * @since 14.7RC1
  */
-@UITest
+@UITest(
+    extraJARs = {
+        // It's currently not possible to install a JAR contributing a Hibernate mapping file as an Extension. Thus
+        // we need to provide the JAR inside WEB-INF/lib. See https://jira.xwiki.org/browse/XWIKI-8271
+        "org.xwiki.platform:xwiki-platform-notifications-filters-default"
+    },
+    resolveExtraJARs = true
```

Comment:
Not specifically related to your changes, is it?

Reply:
No, it's just that without it the generated instance is unusable so I tough I would fix that.